libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -287,4 +393,63 @@ export class OrchestratorAdaptiveRecognizer
             this._resolver = OrchestratorAdaptiveRecognizer.orchestrator.createLabelResolver(snapshot);
         }
     }
+
+    private async tryScoreEntities(text: string, recognizerResult: RecognizerResult) {
+        if (!this.scoreEntities) {
+            return;
+        }
+
+        const results = await this._resolver.score(text, LabelType.Entity);
+        if (!results) {
+            throw new Error(`Failed scoring entities for: ${text}`);
+        }
+
+        // Add full entity recognition result as a 'result' property
+        recognizerResult[this.entityProperty] = results;
+        if (results.length) {
+            if (recognizerResult.entities === null) {
```

Comment:
`recognizerResult.entities ??= {};` can replace this whole block

Reply:
ok